test/flowControlTest/README.md

```diff
@@ -476,5 +482,7 @@ although the notifications queue at this point was 15 (higher than the target by
 ~20 seconds for the response (instead of 48 seconds).
 
 (*) Note that although each notification fails for the same reason (timeout) you will see only one WARN message in the
-logs. This is due to this WARN messge raises a notification alarm for `localhost:1028/givemeDelay` and Orion by default doesn't
-re-log alarm conditions for alarms already risen (except if `-relogAlarms` is used).
+logs. This is due to this WARN message raises a notification alarm for `localhost:1028/givemeDelay` and Orion by default doesn't
```

Comment:
This tutorial was written 3 years ago. One year ago PR https://github.com/telefonicaid/fiware-orion/pull/3970 was merged, implementing re-login in DEBUG tracelevels.

NTC (informative).